tests/unit_tests/lcls_tools/common/data/test_archiver.py

```diff
@@ -613,13 +676,14 @@ def side_effect(pv_list, time_stamp):
 
         mocked_get_data.side_effect = side_effect
 
+        actual_result = get_data_with_time_interval(
+            self.pv_lst,
+            self.time - timedelta(days=10),
+            self.time,
+            timedelta(days=1),
+        )
```

Comment:
We added this piece of code in to help with debugging (so that we could use the debugger to compare `actual_result` to `expected_time_delta_result`). We might have done this is another test or two too. Do we want to keep these for future debugging or delete them since they're not necessary for the test?

Reply:
Hmm. I think developer's choice! I don't have a strong opinion either way

Reply:
I'll just leave it then! I liked it for debugging